measuring-client/src/main/java/de/cyface/app/ui/button/DataCapturingButton.java

```diff
@@ -481,64 +482,103 @@ public void timedOut() {
+                                setButtonStatus(button, PAUSED);
+                                setButtonEnabled(button);
+                                Toast.makeText(context, R.string.toast_measurement_paused, Toast.LENGTH_SHORT).show();
+                            }
+                        }
+                    });
+
+            // Workaround for flaky `rebind` on Android 13+ [RFR-246]
+            // - Don't wait for `shutDownFinished` to be called (flaky due to the bug).
+            // - Use a static 500ms delay to give the measurement some time to stop.
+            if (Build.VERSION.SDK_INT >= 33) {
+                setButtonStatus(button, PAUSED); // avoids button to flip to "stopped" before "paused"
+                final var timeoutHandler = new Handler(context.getMainLooper());
+                timeoutHandler.postAtTime(() -> {
                     // The measurement id should always be set [STAD-333]
-                    Validate.isTrue(measurementIdentifier != -1, "Missing measurement id");
-                    setButtonStatus(button, PAUSED);
+                    // Validate.isTrue(measurementIdentifier != -1, "Missing measurement id");
+                    // Avoids button to flip to "stopped" before "paused"
+                    // setButtonStatus(button, PAUSED);
                     setButtonEnabled(button);
                     Toast.makeText(context, R.string.toast_measurement_paused, Toast.LENGTH_SHORT).show();
-
-                    // Also pause CameraService
-                    // TODO: this way we don't notice when the camera was stopped unexpectedly
-                    cameraService.isRunning(IS_RUNNING_CALLBACK_TIMEOUT, TimeUnit.MILLISECONDS,
-                            new IsRunningCallback() {
-                                @Override
-                                public void isRunning() {
-                                    cameraService.pause(
-                                            new ShutDownFinishedHandler(
-                                                    de.cyface.camera_service.MessageCodes.LOCAL_BROADCAST_SERVICE_STOPPED) {
-                                                @Override
-                                                public void shutDownFinished(long measurementIdentifier) {
-                                                    Log.d(TAG, "pauseCapturing: CameraService stopped");
-                                                }
-                                            });
-                                }
-
-                                @Override
-                                public void timedOut() {
-                                    Log.d(Constants.TAG, "pauseCapturing: no CameraService running, nothing to do");
-                                }
-                            });
-                }
-            });
+                }, SystemClock.uptimeMillis() + 500L);
+            }
```

Comment:
On Android 13+ we don't wait for the `ShutdownFinished` callback:
- when capturing for > ~22 minutes in the background `rebind()` is not called anymore by the system as it should and as it usually is, e.g. on Android <13 or on 13 when the capturing is in the background for < 20 minutes.
- without `rebind`, "0 callers (DCS) are informed about the shutdown finished" by the `BackgroundDataCapturingService`